<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiLLMAgentDockWidgetBase</class>
 <widget class="QDockWidget" name="MultiLLMAgentDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-LLM QGIS Agent</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Comando em Linguagem Natural:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCommand">
      <property name="placeholderText">
       <string>Ex: Adicione um buffer de 100m na camada ativa...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSend">
      <property name="text">
       <string>Gerar Código</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Código Python Gerado:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditCode">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonExecute">
      <property name="text">
       <string>Executar no QGIS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Configurações:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Provedor:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProvider">
        <item>
         <property name="text">
          <string>groq</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>qwen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>openai</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Chave API:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditApiKey">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonTestConnection">
      <property name="text">
       <string>Testar Conexão</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
